test/files/sample.go-nocompile

```diff
@@ -189,6 +202,61 @@ func NewBoozleDefault(typ Types, v int) Boozle {
 	}
 }
 
+func (o Boozle) DeepCopy() Boozle {
+	return Boozle {
+		Typ__ : o.Typ__.DeepCopy(),
+		Bozo__ : (func (x *int) *int {
+			if x == nil {
+				return nil
+			}
+			tmp := (*x)
+			return &tmp
+		})(o.Bozo__),
+		Bippy__ : (func (x *string) *string {
+			if x == nil {
+				return nil
+			}
+			tmp := (*x)
+			return &tmp
+		})(o.Bippy__),
+		Aggle__ : (func (x *[]int) *[]int {
+			if x == nil {
+				return nil
+			}
+			tmp := (func (x []int) []int {
+				var ret []int
+				for _, v := range x {
+					v = v
+					ret = append(ret, v)
+				}
```

Comment:
This case is handled.  In this particular example, we know `x` is a simple data type and doesn't need copying.  But look at `Wow` for an example where it does